templates/integrations.html

```diff
@@ -0,0 +1,179 @@
+    <div class="row--50-50">
+      <div class="col">
+        <h1 class="p-heading--2">How do integrations work?</h1>
+      </div>
+      <div class="col"></div>
+    </div>
+  </section>
+  <section class="p-section--shallow">
+    <div class="row--50-50">
+      <div class="u-fixed-width"><hr class="p-rule--muted"></div>
+      <div class="col">
+        {{ image(
+            url="https://assets.ubuntu.com/v1/cc5b7188-Integrations%201.png",
+            alt="",
+            height="173",
+            width="600",
+            hi_def=True,
+            loading="auto",
+          ) | safe
+        }}
+      </div>
+      <div class="col">
+        <p>All charms contain information about the services they offer or require to operate correctly. You can connect any two applications that:</p>
+        <div class="col-start-large-7">
+          <hr class="p-rule--muted">
+        </div>
+        <ul class="p-list--divided">
+          <li class="p-list__item has-bullet">Have compatible <strong>endpoints</strong> (i.e. requirer with provider, or peer with peer)</li>
+          <li class="p-list__item has-bullet">Implement the same <strong>interface</strong> (i.e. exchange the same set of information)</li>
+        </ul>
+      </div>
+    </div>
+  </section>
+  <section class="p-section--shallow">
+    <div class="row--50-50">
+      <div class="u-fixed-width"><hr class="p-rule--muted"></div>
+      <div class="col">
+        {{ image(
+            url="https://assets.ubuntu.com/v1/a1c6a3fd-Integrations%202.png",
+            alt="",
+            height="340",
+            width="600",
+            hi_def=True,
+            loading="auto",
+          ) | safe
+        }}
+      </div>
+      <div class="col">
+        <p>When your model contains two charms you wish to connect, all you have to do is to run juju integrate from the Juju controller and specify the two charms to be connected.</p>
+      </div>
+      <div class="u-fixed-width"><hr class="p-rule--muted"></div>
+    </div>
+  </section>
+  <section class="p-section--deep">
+    <div class="row--50-50">
+      <div class="col">
+        {{ image(
+            url="https://assets.ubuntu.com/v1/39f6c093-integrations%203.png",
+            alt="",
+            height="173",
+            width="600",
+            hi_def=True,
+            loading="auto",
+          ) | safe
+        }}
+      </div>
+      <div class="col">
+        <p>Juju will automatically create an integration between these two services. You generally do not need to worry about anything else.</p>
+        <p>The integrated applications communicate directly with each other and the communication is not mediated by the juju controller.</p>
+      </div>
+    </div>
+  </section>
+  <section class="p-section--shallow">
+    <div class="row--50-50">
+      <div class="u-fixed-width"><hr class="p-rule--muted"></div>
+      <div class="col">
+        <div style="padding-left: 2rem; padding-top: 2rem;">
```

Comment:
Not sure why padding left is needed?

But again, top padding could be moved to shallow section in previous element.